### ISSUE TITLE
[FR] Add mass to the variables

### ISSUE THREAD

**Issue by mmikhasenko** [KaiHabermann/decayangle#73]
With the `mass` provided next to `phi` and `theta` for every vertex, the representation will be complete

**Comment by KaiHabermann** [comment]
Yes we could do that indeed. Maybe such that people can disable it?

**Comment by mmikhasenko** [comment]
Nice!!
there is no reason to disable, it's a dictionary, - take what ever you want.

**Comment by KaiHabermann** [comment]
I made a merge request for this change. It changes how you would automatically unpack the angles though.
